src/arg-parser.js

```diff
@@ -11,6 +13,13 @@ const parse = argv => {
       acc.quiet = true;
     }
 
+    if (isHash(arg)) {
```

Comment:
Wondering if it's safe enough to just take any argument which does not with a dash as a hash? That way branches and tags e.t.c would also be supported? What are your thoughts? Not a deal breaker or anything, appreciate it suits requirements fine as is :)

Reply:
@lhaggar Sorry, I don't understand this comment. Git will always send hashes, could you please elaborate?

Reply:
@michaellopez Husky will pass hashes but theoretically you could call this module globally or via npx e.t.c and pass it valid git tags or branches, e.g. compare master and a feature branch. Was just a thought (its not a big deal), don't mind if it stays as it is :)

Reply:
Ah, alright, I'll see what I can do 👍